.ci.yaml

```diff
@@ -2084,6 +2084,16 @@ targets:
       task_name: picture_cache_perf__timeline_summary
     scheduler: luci
 
+  - name: Linux_android picture_cache_complexity_scoring_perf__timeline_summary
```

Comment:
Could we add `bringup: true` for both tests? Flakiness needs to get validated in staging pool first before enabled in prod.